03-amazon_s3.Rmd

```diff
@@ -0,0 +1,244 @@
+3.  Select __Create a new warehouse data source__.
+4.  Enter a name for the warehouse data source -- this must be prefixed with '-alpha'.
+5.  Select __Create__.
+
+When you create a new warehouse data source, only you will initially have access. As an admin of the data source, you will be able to add and remove other users from the data access group as required. Further information on managing data access groups can be found in Section \@ref(manage-access-to-a-bucket).
+
+### Access levels
+
+Every bucket has three access levels:
+
+*   Read only
+*   Read/write
+*   Admin
+
+### Request access to a bucket
+
+To gain access to a bucket (warehouse data source or app data source), you must be added to the relevant data access group.
+
+If you know an admin of the bucket you require access to, you can ask them to add you to the data access group.
+
+If you do not know any of the admins of the bucket you require access to, you can ask the Analytical Platform team to add you to the data access group on the [#ap_admin_request](https://asdslack.slack.com/messages/CBLAGCQG6/) Slack channel or by [email](mailto:analytical_platform@digital.justice.gov.uk).
+
+When requesting access to a bucket, you should specify the name of the bucket and the level of access you require.
+
+### Manage access to a bucket{#manage-access-to-a-bucket}
+
+Bucket admins can currently only manage access to warehouse data sources in the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz/). You cannot manage access to buckets directly in the Amazon S3 console.
+
+To manage access to a warehouse data source:
+
+1.  Go to the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Warehouse data__ tab.
+3.  Select the name of the warehouse data source you want to manage.
+
+To add a new user to the data access group:
+
+1.  Type the user's GitHub username into the input field labelled __Grant access to this data to other users__.
+2.  Select the user from the drop-down list.
+3.  Select __Grant access__.
+
+To edit the access level of a user:
+
+1.  Select __Edit access level__ next to the name of the user.
+2.  Select the checkbox next to the access level you wish to grant the user.
+3.  Select __Save__.
+
+To remove a user from the data access group:
+
+1.  Select __Edit access level__ next to the name of the user.
+2.  Select __Revoke access__.
+
+## Upload files to Amazon S3
+
+You can upload files to Amazon S3 from your local computer or from RStudio or JupyterLab.
+
+When uploading files to Amazon S3, you should ensure that you follow all necessary [information governance]() procedures. In particular, you must complete a data movement form when moving any data onto the Analytical Platform.
+
+### Amazon S3 Console
+
+You can use the Amazon S3 Console to upload files from your local computer (e.g., personal or shared storage on DOM1 or Quantum) only.
```

Comment:
```suggestion
You can use the Amazon S3 Console to upload files from your local computer (e.g. personal or shared storage on DOM1 or Quantum) only.
```

Reply:
Changed to 'for example' throughout in line with GDS style guide. (26514c9)